ios/Plugin/Plugin.swift

```diff
@@ -89,6 +89,16 @@ public class SentryCapacitor: CAPPlugin {
         call.resolve()
     }
 
+    @objc func getStringBytesLength(_ call: CAPPluginCall) {
```

Comment:
Is this called from JS for both iOS and Android?

Reply:
Yes, but only Android implemented it 
https://github.com/getsentry/sentry-capacitor/blob/ec773dd310ae932b8eecf62c09a42c4a26e306c4/android/src/main/java/io/sentry/capacitor/SentryCapacitor.java#L239

in this case on iOS it was falling back to the value set in JS, which worked for most cases but not all of them.